README.md

```diff
@@ -50,18 +50,24 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
 
 ## Installation
 
+The package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
+
 ### Dependencies
 
-Before installing deeprank2 you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.9 installed. Follow the official documentation linked below:
+Before installing deeprank2 you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.10 installed. The following dependency installation instructions are updated as of 14/09/2023, but in case of issues during installation always refer to the official documentation which is linked below:
 
-*  [MSMS](https://ssbio.readthedocs.io/en/latest/instructions/msms.html)
-  *  [Here](https://ssbio.readthedocs.io/en/latest/instructions/msms.html) for MacOS with M1 chip users
+*  [MSMS](https://anaconda.org/bioconda/msms): `conda install -c bioconda msms`.
+    *  [Here](https://ssbio.readthedocs.io/en/latest/instructions/msms.html) for MacOS with M1 chip users.
 *  [PyTorch](https://pytorch.org/get-started/locally/)
-  *  We support torch's CPU library as well as CUDA
-*  [PyG](https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html) and its optional dependencies: `torch_scatter`, `torch_sparse`, `torch_cluster`, `torch_spline_conv`
+    *  We support torch's CPU library as well as CUDA.
+*  [PyG](https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html) and its optional dependencies: `torch_scatter`, `torch_sparse`, `torch_cluster`, `torch_spline_conv`.
```

Comment:
you want to give the command for these as well? Just so users don't have to scroll the external sites to find it?

Reply:
Since both pytorch and pytorch geom have the nice table which the user can set according to the OS, and it gets update quite often, I'd leave to the user to find the commands, unless at a certain point it becomes necessary because a specific version is needed (right now all is working with the latest torch versions).